spec/triggers/new_event_spec.rb

```diff
@@ -32,8 +32,8 @@
     end
 
     it 'payload and data start with https://' do
-      expect(output[:data]).to include(:callback)
-      expect(output[:data][:callback]).to start_with 'https://'
+      puts output[:data]
```

Comment:
@klaasvandeweerdt Should we remove this?

Reply:
@bennettgo, yes, apologies for that mistake, that was for debugging purposes. This will be removed in the next commit.